src/MSONParameterParser.h

```diff
@@ -178,6 +181,8 @@ namespace snowcrash {
                     }
                     else {
                         if (RegexMatch(attributes[i], "^enum\\[[^][]+]$")) {
+
+                            // Retrieve the type which is wrapped by enum[]
                             out.node.type = attributes[i].substr(5, attributes[i].length() - 6);
```

Comment:
I think it wold be good if it is tolerant to white spaces. Also please replace the numbers with some constants + comments. We really had no idea whats up reading these numbers.